app/components/find/results/search_result_component.html.erb

```diff
@@ -6,8 +6,19 @@
     <% end %>
   </h2>
   <dl class="app-description-list app-description-list--search-result">
-    <dt class="app-description-list__label">Study type</dt>
-    <dd data-qa="course__study_mode"><%= course.study_mode.humanize %></dd>
+
+    <dt class="app-description-list__label">Fee or salary</dt>
+    <dd data-qa="course__funding_options"><%= course.funding %>
+      <br>
+      <p class="govuk-hint"><%= course.funding_option %>
+    </dd>
+
+    <dt class="app-description-list__label">Course fee</dt>
+    <dd data-qa="course__visa_sponsorship"><%= course_fee_value %></dd>
+
+    <dt class="app-description-list__label">Visa sponsorship</dt>
+    <dd data-qa="course__visa_sponsorship"><%= visa_sponsorship_status %></dd>
```

Comment:
Same as above, just wondering about the `data-qa` value

Reply:
Also updated here: https://github.com/DFE-Digital/publish-teacher-training/pull/4093/commits/459a5dc17ceb8c4ec10bb881696c7098242df3b5